app/assets/stylesheets/application.scss

```diff
@@ -1 +1,8 @@
- @import "bootstrap";
\ No newline at end of file
+@import 'bootstrap';
+.new-button {
```

Comment:
css class should not be created in here. it should be in another separate file, then be included in the `application.scss`